source/blog/2017-02-10-matching-fedora-ostree-released-content-with-each-2week-atomic-release.html.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Matching Fedora OSTree Released Content With Each 2 Week Atomic Release' 
+author: dustymabe
+date: 2017-02-10 15:23:32 UTC
+tags: fedora, atomic
+published: true
+comments: true
+---
+
+In Fedora we release a new Atomic Host approximately every two weeks. In the
+past this has meant that we bless and ship new ISO, QCOW, and Vagrant images that
+can then be used to install and or start a new Atomic Host server. But
+what if you already have an Atomic Host server up and running? 
+
+READMORE
+
+Servers that are already running are configured to get their updates
+directly from the OSTree repo that is sitting on Fedora Infrastructure
+servers. The client will ask "What is the newest commit for my
+branch/reference?" and the server will kindly reply with the most recent commit.
+If the client is at an older version then it will start to pull the
+newer commit and will apply the update. 
+
+This is exactly how the client is supposed to behave, but one problem
+with the way we have been doing things in the past is that we have
+been updating everyones "branch/reference" every night when we do
+our updates runs in Fedora.
+
+This has the side effect of meaning that users can get content as soon
+as it has been created, but it also means that the two week release
+process where we perform testing and validation really means nothing
+for these users, as they will get something before we ever did testing
+on it. 
```

Comment:
I'd replace all three of these paragraphs with something like my suggested paragraph above.